pkg/cmd/deletecmd/delete_preview.go

```diff
@@ -116,7 +116,15 @@ func (o *DeletePreviewOptions) DeletePreview(name string) error {
 		return err
 	}
 
-	environment, err := kube.GetEnvironment(jxClient, ns, name)
+	kubeClient, err := o.KubeClient()
```

Comment:
Would it not be easier to pass the dev namespace into `DeletePreview`? Or probably even better the `Environment` directly. Looking at the callers of this method one pretty much has in all cases the environment already, no need to re-get it (especially if one changes the impl of GetFilteredEnvironmentNames)?

Reply:
I'll take a look, thanks @hferentschik